src/components/productCard/styles.css

```diff
@@ -0,0 +1,99 @@
+.cardDescription,
+.cardRating,
+.cardPrice {
+  font-family: "Roboto";
```

Comment:
Todo app é Roboto? Se for, vale a pena definir a font-family de maneira global. Outro ponto, é sempre bom colocar o sans-serif, pois caso o navegador não consiga usar a Roboto ele irá aplicar uma font sem serifa disponível

`font-family: 'Roboto', sans-serif;`